firestore/firestore_quickstart/main.go

```diff
@@ -16,6 +16,7 @@
 package main
 
 // [START fs_initialize]
+// [START firestore_setup_client_create]
```

Comment:
Aren't, the new tags should be around the old ones? When the code will be merged, these new tags will be shown in documentation.

Reply:
It was BenWhitehead's proposal. I already pushed PR